python/python/glide/async_commands/core.py

```diff
@@ -2326,6 +2326,58 @@ async def zscore(self, key: str, member: str) -> Optional[float]:
             await self._execute_command(RequestType.ZScore, [key, member]),
         )
 
+    async def zdiff(self, keys: List[str]) -> List[str]:
+        """
+        Returns the difference between the first sorted set and all the successive sorted sets.
+        To get the elements with their scores, see `zdiff_withscores`.
+
+        See https://redis.io/commands/zdiff/ for more details.
+
```

Comment:
please add same slot notice

Reply:
Good point, note that his notice is missing for the sorted set commands in Java so they will need to be updated

Reply:
See #226